### ISSUE TITLE
when autoreconf -i  ,some warning reported

### ISSUE THREAD

**Issue by availa** [aria2/aria2#1066]
/src/aria2 # autoreconf -i
aclocal-1.15: overwriting 'm4/ax_check_compile_flag.m4' with '/usr/share/aclocal/ax_check_compile_flag.m4'
aclocal-1.15: overwriting 'm4/ax_cxx_compile_stdcxx_11.m4' with '/usr/share/aclocal/ax_cxx_compile_stdcxx_11.m4'
configure.ac:731: warning: The 'AM_PROG_MKDIR_P' macro is deprecated, and its use is discouraged.
configure.ac:731: You should use the Autoconf-provided 'AC_PROG_MKDIR_P' macro instead,
configure.ac:731: and use '$(MKDIR_P)' instead of '$(mkdir_p)'in your Makefile.am files.

**Comment by tatsuhiro-t** [comment]
Yes, it is known issue.  It looks like a third party library uses the deprecated macro.

**Comment by LamCiuLoeng** [comment]
so how to fix ?

**Comment by JanthonAlyn** [comment]
> so how to fix ?

Edit the file "configure.ac" in src folder.
in line 740
AM_GNU_GETTEXT_VERSION([0.18])   ==>  AM_GNU_GETTEXT_VERSION([0.19])
![图片](https://github.com/aria2/aria2/assets/59256720/c532980b-c409-4782-91e2-a39c0746cafa)
